lua/tardis/interiors/templates/ttcapsule.lua

```diff
@@ -35,7 +35,15 @@ T = {
                 angoffset=Angle(0,0,0)
             },
         }
-    }
+    },
+    Interior = {
```

Comment:
This appears to have broken the default interior TT version with the interior door backfaces now being invisible
![image](https://user-images.githubusercontent.com/2363642/154178560-5a8dfa38-0720-4757-8274-4431c7e2114c.png)

Reply:
Fixed